Anna_Gamzinova/MongoDB/objects/base_db.py

```diff
@@ -0,0 +1,34 @@
+import pymongo
+
+
+class BaseDB:
+    def __init__(self, database, collection):
+        self.connection = pymongo.MongoClient("mongodb://127.0.0.1:27017/")
+        self.database = self.connection[database]
+        self.collection = self.database[collection]
+
+    def insert_value(self, new_name, new_color, new_occupation):
+        """
+        dict format is {'name': new_name, 'color': new_color, 'occupation': new_occupation}
+        """
+        return self.collection.insert_one({'name': new_name, 'color': new_color, 'occupation': new_occupation})
+
+    def insert_values(self, my_tuples: tuple):
+        """
+        list should be filled with dicts, with such format {'name': new_name, 'color': new_color, 'occupation':
+        new_occupation}
+        """
+        return self.collection.insert_many(
+            [{'name': insert_i[0], 'color': insert_i[1], 'occupation': insert_i[2]} for insert_i in my_tuples])
+
+    def find_value(self):
+        return self.collection.find_one()
```

Comment:
find_one works with filter. You can pass dict to it to find specific value

Reply:
fixed